### PR TITLE
Fixes #1266 populate mononuclear phagocyte

### LINKED ISSUE #1266: [Typo/Bug] mononuclear & multinucleate phagocyte have no children!
**CL term**

mononuclear phagocyte  http://purl.obolibrary.org/obo/CL_0000113 

**Description of typo, bug or error**

image

@addiehl - any suggestions for how we might efficiently populate these classes?

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #1266 populate mononuclear phagocyte

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by aleixpuigb** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -4300,6 +4300,7 @@ AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0000092 "chondroclast")
 AnnotationAssertion(rdfs:comment obo:CL_0000092 "Morphology: Highly vesicular; markers: Surface: RANK, cFMS (MCSF receptor); Secreted: cathepsin K and TRAP (tartate resistant acid phosphatase); transcription factors: PU.1, cFOS, MITF, NFkB (p52); role or process: tissue remodelling: bone resorption; lineage: hematopoietic, myeloid.")
 AnnotationAssertion(rdfs:label obo:CL_0000092 "osteoclast")
 EquivalentClasses(obo:CL_0000092 ObjectIntersectionOf(obo:CL_0000766 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001474) ObjectSomeValuesFrom(obo:BFO_0000051 obo:PR_000001850) ObjectSomeValuesFrom(obo:BFO_0000051 obo:PR_000001937) ObjectSomeValuesFrom(obo:BFO_0000051 obo:PR_000001944) ObjectSomeValuesFrom(obo:BFO_0000051 obo:PR_000007597) ObjectSomeValuesFrom(obo:BFO_0000051 obo:PR_000011178) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001954) ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000002062) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0045453)))
+SubClassOf(obo:CL_0000092 obo:CL_0000518)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:4303]: Did not see this edit reflected in tasks of related ticket.

**Review by bvarner-ebi — COMMENTED** [review]
Edits do not appear to match tasks described in linked ticket.

This [comment](https://github.com/obophenotype/cell-ontology/issues/1266#issuecomment-1781343956) suggests 'dendritic cell' and monocyte will be classified as children of 'mononuclear phagocyte'. 
See also inline comment.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**